### DISCUSSION TITLE
Adonis Model Audit

### DISCUSSION THREAD

**Discussion by manfe** [adonisjs/core#3531] [General]
## Auditable Model

I was a ruby on rails developer and I found adonisjs recently and I am building an API with it. I am enjoying it. But I would like to know if there is any auditable package recommend by core team about it, such as ruby gem called [audited](https://github.com/collectiveidea/audited)

Thank you for all the work done.

Appreciate ❤️

**Comment by berstanislaw** [comment]
:+1:  +1

**Comment by jeanbispo** [comment]
In version 4 we used this lib, however the original author stopped supporting it and has not yet been ported to adonis 5. 
https://github.com/ahtinurme/adonis-audit

**Comment by jeanbispo** [comment]
#3316
